bgpd/bgp_route.c

```diff
@@ -1781,6 +1781,10 @@ int subgroup_announce_check(struct bgp_node *rn, struct bgp_path_info *pi,
 		peer->rmap_type = 0;
 
 		if (ret == RMAP_DENYMATCH) {
+			if (bgp_debug_update(NULL, p, subgrp->update_group, 0))
```

Comment:
The changes for the lib/routemap.c code are sufficient enough that this needs to be in a different commit.

Reply:
Okay..I will open another pull request for this.